DOCS/contribute.md

```diff
@@ -15,8 +15,8 @@ Sending patches
   ``git format-patch``.
 - Plain diffs posted as pastebins are not acceptable! (Especially if the http
   link returns HTML.) They only cause extra work for everyone, because they lack
-  commit message and authorship information.
-- Never send patches to any of the developers email addresses.
+  commit messages and authorship information.
```

Comment:
wouldn't it "authorship informations" too?

Reply:
apparently that is a german thing and there is no plural of information in english.

Reply:
There is no plural for uncountable noun